src/pconn.cc

```diff
@@ -348,33 +350,30 @@ PconnPool::key(const Comm::ConnectionPointer &destLink, const char *domain)
 }
 
 void
-PconnPool::dumpHist(StoreEntry * e) const
+PconnPool::dumpHist(std::ostream &yaml) const
 {
-    storeAppendPrintf(e,
-                      "%s persistent connection counts:\n"
-                      "\n"
-                      "\t Requests\t Connection Count\n"
-                      "\t --------\t ----------------\n",
-                      descr);
+    AtMostOnce title("  persistent connection counts:\n");
+    IfNoOutput defaultText("  {}\n", yaml);
```

Comment:
We do not need to report the histogram if there are no connections.

```suggestion
```

Reply:
The problem I'm trying to address is one where there is no output at all, which would be confusing
But if we output even only just "open connections count: 0", that is addressed

Reply:
> The problem I'm trying to address is one where there is no output at all, which would be confusing But if we output even only just "open connections count: 0", that is addressed

Exactly. As you know, I would not add the word "count" -- a number that follows makes it obvious that this is a "number of X" or "X count" statistics.

Reply:
removed